src/com/sap/piper/integration/TransportManagementService.groovy

```diff
@@ -51,9 +51,14 @@ class TransportManagementService implements Serializable {
         }
 
         def response = sendApiRequest(parameters)
+
+        if (config.verbose) {
+            echo("Received response with status ${response.status} from authentification request.")
```

Comment:
Don't you have to call `echo()` on the `script` instance?

Reply:
No, not in this case. There is a `echo` method provided here in this class calling in turn `script.echo`.